### PR TITLE
Displays a message at startup

### PR DESCRIPTION
**Pull request by Giwi**
Displays a message at startup with WARP10_HEAP, WARP10_HEAP_MAX, maxfetch and maxops

### CODE REVIEW

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 1]
```diff
@@ -560,6 +561,13 @@ public static void main(String[] args) throws Exception {
     }
 
     WarpDist.setInitialized(true);
+    System.out.println("## Your Warp 10 setup:");
+    System.out.printf("## - WARP10_HEAP:              %s\n", FileUtils.byteCountToDisplaySize(Runtime.getRuntime().totalMemory()));
+    System.out.printf("## - WARP10_HEAP_MAX:          %s\n", FileUtils.byteCountToDisplaySize(Runtime.getRuntime().maxMemory()));
+    System.out.printf("## - warpscript.maxfetch:      %s\n", properties.getProperty("warpscript.maxfetch"));
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/standalone/Warp.java:567]: Please use parameter name from Configuration.java.

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 2]
```diff
@@ -560,6 +561,13 @@ public static void main(String[] args) throws Exception {
     }
 
     WarpDist.setInitialized(true);
+    System.out.println("## Your Warp 10 setup:");
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/standalone/Warp.java:564]: Why stdout when most of stdout outputs have been replaced by logging calls?

Why those specific parameters? What is the purpose of these outputs?

**Review by hbs — CHANGES_REQUESTED** [review]

File: warp10/src/main/java/io/warp10/standalone/Warp.java (hunk 1 above)
Comment (Giwi) [reply warp10/src/main/java/io/warp10/standalone/Warp.java:567]: indeed

File: warp10/src/main/java/io/warp10/standalone/Warp.java (hunk 2 above)
Comment (Giwi) [reply warp10/src/main/java/io/warp10/standalone/Warp.java:564]: The purpose of this PR is to display some important metrics for platform sizing like the JVM heap or max fetch and ops.

**Comment by pi-r-p** [comment]
Printing these values at startup make sense if we propose by default relative RAM settings 
 -XX:MinRAMPercentage=70.0  (for small systems, less than 100MB of ram)
 -XX:MaxRAMPercentage=50.0 (for others)

It is supported since Java 8u191 (October 16, 2018)

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 3]
```diff
@@ -16,50 +16,15 @@
...
 import io.warp10.continuum.Configuration;
 import io.warp10.continuum.JettyUtil;
 import io.warp10.continuum.ThrottlingManager;
-import io.warp10.continuum.egress.CORSHandler;
-import io.warp10.continuum.egress.EgressExecHandler;
-import io.warp10.continuum.egress.EgressFetchHandler;
-import io.warp10.continuum.egress.EgressFindHandler;
-import io.warp10.continuum.egress.EgressInteractiveHandler;
-import io.warp10.continuum.egress.EgressMobiusHandler;
+import io.warp10.continuum.egress.*;
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/standalone/Warp.java:27]: Please avoid wildcard imports

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 4]
```diff
@@ -69,11 +34,32 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.*;
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/standalone/Warp.java:61]: Ditto

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 5]
```diff
@@ -97,17 +83,17 @@ public class Warp extends WarpDist implements Runnable {
...
-    Configuration.WARP_HASH_CLASS,
-    Configuration.WARP_HASH_LABELS,
-    Configuration.WARP_HASH_TOKEN,
-    Configuration.WARP_HASH_APP,
-    Configuration.WARP_AES_TOKEN,
-    Configuration.WARP_AES_SCRIPTS,
-    Configuration.CONFIG_WARPSCRIPT_UPDATE_ENDPOINT,
-    Configuration.CONFIG_WARPSCRIPT_META_ENDPOINT,
-    Configuration.WARP_TIME_UNITS,
+      Configuration.INGRESS_WEBSOCKET_MAXMESSAGESIZE,
```
Comment (hbs) [inline warp10/src/main/java/io/warp10/standalone/Warp.java:86]: Indent is 2 whitespaces

Comment (Giwi) [reply warp10/src/main/java/io/warp10/standalone/Warp.java]: done

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 6]
```diff
@@ -560,6 +558,13 @@ public static void main(String[] args) throws Exception {
     }
 
     WarpDist.setInitialized(true);
+    LOG.info("## Your Warp 10 setup:");
+    LOG.info("## - WARP10_HEAP:              " + FileUtils.byteCountToDisplaySize(Runtime.getRuntime().totalMemory()));
+    LOG.info("## - WARP10_HEAP_MAX:          " + FileUtils.byteCountToDisplaySize(Runtime.getRuntime().maxMemory()));
+    LOG.info("## - " + Configuration.WARPSCRIPT_MAX_FETCH + ":      " + properties.getProperty(Configuration.WARPSCRIPT_MAX_FETCH));
+    LOG.info("## - " + Configuration.WARPSCRIPT_MAX_FETCH_HARD + ": " + properties.getProperty(Configuration.WARPSCRIPT_MAX_FETCH));
```
Comment (steven-gueguen) [inline warp10/src/main/java/io/warp10/standalone/Warp.java:565]: Should be `Configuration.WARPSCRIPT_MAX_FETCH_HARD`

**Review by steven-gueguen — CHANGES_REQUESTED** [review]

**Review by steven-gueguen — APPROVED** [review]

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 7]
```diff
@@ -154,7 +152,7 @@ public static void main(String[] args) throws Exception {
     boolean enableStreamUpdate = !("true".equals(properties.getProperty(Configuration.WARP_STREAMUPDATE_DISABLE)));
     boolean enableREL = !("true".equals(properties.getProperty(Configuration.WARP_INTERACTIVE_DISABLE)));
 
-    for (String property: REQUIRED_PROPERTIES) {
+    for (String property : REQUIRED_PROPERTIES) {
```
Comment (randomboolean) [inline warp10/src/main/java/io/warp10/standalone/Warp.java:155]: Revert to usual syntax used in the code base.
Ditto for lines #177,179,201,421,429

**Review by hbs — APPROVED** [review]

**Pull request merged**